maintainers/maintainer-list.nix

```diff
@@ -2248,6 +2248,12 @@
     githubId = 411447;
     name = "Leo Gaspard";
   };
+  ekpdt = {
+    email = "nix@pdtpartners.com";
+    github = "ekpdt";
+    githubId = 64796365;
+    name = "Elan Kugelmass";
```

Comment:
Please put this into a separate commit.

Reply:
Same PR, separate commit
or
Different PR
?

Reply:
> Same PR, separate commit